google-codelab-index/google_codelab_index_cards.js

```diff
@@ -164,13 +164,25 @@ class Cards extends HTMLElement {
     console.log('filter');
   }
 
+  /**
+   * 
+   * @param {string} category 
+   * @returns {string}
+   */
+  normalizeCategory_(category) {
+    return category.toLowerCase()
+        .replace(/\s+/g, '-')           // Replace spaces with -
+        .replace(/\-\-+/g, '-')         // Replace multiple - with single -
```

Comment:
This is weird. Normally, `-` outside of square brackets `[ ]` shouldn't count as a special regexp character. In fact, I tried in chrome js console. Both work:

![image](https://user-images.githubusercontent.com/25405/39291587-d0508642-4933-11e8-8f75-778af9c30a3b.png)

What can I say. Weird javascript. :)

Reply:
I probably stuffed up :)